
// this file is automatically generated. you are highly discouraged from
// editing it manually
#include <pblog/pblog.h>

#include <pblog/pblogd.pb.h>
#include <pblog/protocol.pb.h>
#include <pblog/message.pb.h>
#include <pblog/protocol/response.pb.h>
#include <pblog/protocol/request.pb.h>
// @@protoc_insertion_point(includes)

namespace pblog {

    PBLOG_EXPORT void register_known_messages();
}
